### PR TITLE
fix some links in relation to docs nav changes

### PR DESCRIPTION
**Pull request by susanev**
this is fixing up some links in relation to the docs nav work
this should not be merged until after that lands
there's no need for a timed release or anything
redirects are in place and these links will still work 

related pr https://github.com/pulumi/pulumi-hugo/pull/2769

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3712,7 +3712,7 @@ Replaced by 3.33.1 during release process.
 
 ### Breaking Changes
 
-Please note, the following 2 breaking changes were included in our [3.0 changlog](https://www.pulumi.com/docs/get-started/install/migrating-3.0/#updated-cli-behavior-in-pulumi-30)
```
Comment (justinvp) [inline CHANGELOG.md:3715]: It looks like this is the only line the changed. But there are tons of changes in the file. Could you remove all the other (whitespace?) changes in this file so only this one line shows up as a change?

**Review by justinvp — APPROVED** [review]
LGTM otherwise

**Review by RobbieMcKinstry — APPROVED** [review]
I have one question about the diff in the CHANGELOG file. Other than that, looks good to me. Thanks susan! 🤩

File: CHANGELOG.md
[the remark this answers is not in the document]
Comment (RobbieMcKinstry) [reply CHANGELOG.md:1]: Oh! It looks like there's also a merge conflict around the changelog too. Removing the changelog from the diff would resolve the issue.

**Comment by RobbieMcKinstry** [comment]
Also, I think we can remove the DNM label from this PR now that the dependent PR has been merged.

File: CHANGELOG.md (hunk 1 above)
Comment (susanev) [reply CHANGELOG.md]: yah, no clue what i was doing; removed the changes to this file

File: CHANGELOG.md
[the remark this answers is not in the document]
Comment (susanev) [reply CHANGELOG.md:1]: yah, no clue what i was doing; removed the changes to this file

**Comment by susanev** [comment]
bors merge

**Pull request merged**